src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -378,7 +378,6 @@ public partial class SocketException : System.Exception
     {
         public SocketException() { }
         public SocketException(int errorCode) { }
-        public override string Message { get { return default(string); } }
```

Comment:
This is changing the public API surface, isn't it?  Or is this is not considered a change since the .Message property still exists in the base class `Exception`?